eternalcore-core/src/main/java/com/eternalcode/core/viewer/BukkitViewerImpl.java

```diff
@@ -42,6 +43,15 @@ public boolean isConsole() {
         return this.console;
     }
 
+    @Override
+    public String getName() {
+        if (console) {
+            return "CONSOLE";
+        }
+
+        return Bukkit.getServer().getOfflinePlayer(uuid).getName();
```

Comment:
`getOfflinePlayer` jest operacją blokującą zrób getPlayer i jeśli jest null to throw ISE z info, że gracz jest offline